discord/message.py

```diff
@@ -1024,6 +1024,9 @@ def _update(self, data):
                 continue
             else:
                 handler(self, value)
+                # Update the message's _raw_data
+                # ref: #2777
```

Comment:
Do we need the comment?

Reply:
Idk I can remove it

Reply:
yeah, i see no reason why we should keep that :)